[2022-03-17 03:18:52,011] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: testing_dag.ingest 2019-02-02T06:00:00+00:00 [queued]>
[2022-03-17 03:18:52,080] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: testing_dag.ingest 2019-02-02T06:00:00+00:00 [queued]>
[2022-03-17 03:18:52,082] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-03-17 03:18:52,083] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2022-03-17 03:18:52,085] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-03-17 03:18:52,117] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): ingest> on 2019-02-02T06:00:00+00:00
[2022-03-17 03:18:52,123] {standard_task_runner.py:52} INFO - Started process 750 to run task
[2022-03-17 03:18:52,136] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'testing_dag', 'ingest', '2019-02-02T06:00:00+00:00', '--job-id', '9', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/testing_dag.py', '--cfg-path', '/tmp/tmpb8918xls', '--error-file', '/tmp/tmp0n22o1j2']
[2022-03-17 03:18:52,143] {standard_task_runner.py:77} INFO - Job 9: Subtask ingest
[2022-03-17 03:18:52,275] {logging_mixin.py:104} INFO - Running <TaskInstance: testing_dag.ingest 2019-02-02T06:00:00+00:00 [running]> on host 509d5aa4e827
[2022-03-17 03:18:52,384] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=testing_dag
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2019-02-02T06:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2019-02-02T06:00:00+00:00
[2022-03-17 03:18:52,388] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2022-03-17 03:18:52,392] {bash.py:158} INFO - Running command: ls /opt/airflow
[2022-03-17 03:18:52,405] {bash.py:169} INFO - Output:
[2022-03-17 03:18:52,482] {bash.py:173} INFO - airflow.cfg
[2022-03-17 03:18:52,484] {bash.py:173} INFO - dags
[2022-03-17 03:18:52,485] {bash.py:173} INFO - google_credentials.json
[2022-03-17 03:18:52,486] {bash.py:173} INFO - logs
[2022-03-17 03:18:52,488] {bash.py:173} INFO - output_2019-01.csv
[2022-03-17 03:18:52,488] {bash.py:173} INFO - output_2019-02.csv
[2022-03-17 03:18:52,490] {bash.py:173} INFO - output_2019-03.csv
[2022-03-17 03:18:52,491] {bash.py:173} INFO - plugins
[2022-03-17 03:18:52,492] {bash.py:173} INFO - unittests.cfg
[2022-03-17 03:18:52,493] {bash.py:173} INFO - webserver_config.py
[2022-03-17 03:18:52,494] {bash.py:177} INFO - Command exited with return code 0
[2022-03-17 03:18:52,543] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=testing_dag, task_id=ingest, execution_date=20190202T060000, start_date=20220317T031852, end_date=20220317T031852
[2022-03-17 03:18:52,598] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-03-17 03:18:52,627] {local_task_job.py:146} INFO - Task exited with return code 0
